### PR TITLE
added a step to shorten a url using the firebase api

### PR DESCRIPTION
**Pull request by alexisandreason**

### CODE REVIEW

**Review by matthewtsmith — CHANGES_REQUESTED** [review]
@alexisandreason Code looks good but I think this should be in the firebase package instead of convert.

File: convert_pkg/package.yaml [hunk 1]
```diff
@@ -9,14 +9,20 @@ executables:
...
     inputs:
-      XML:
+      FirebaseApiKey:
         required: true
-        description: XML you'd like to parse in a string
+        description: Firebase api key used for authentication
+      FirebaseUrlPrefix:
+        required: true
+        description: The url prefix from firebase
+      UrlToShorten:
```
Comment (matthewtsmith) [inline convert_pkg/package.yaml]: Change to just `url`

File: convert_pkg/package.yaml [hunk 2]
```diff
@@ -9,14 +9,20 @@ executables:
...
 steps:
-  parse_xml_to_object@1.0:
+  shorten_url@1.0:
     inputs:
-      XML:
+      FirebaseApiKey:
         required: true
-        description: XML you'd like to parse in a string
+        description: Firebase api key used for authentication
+      FirebaseUrlPrefix:
```
Comment (matthewtsmith) [inline convert_pkg/package.yaml]: Provide a `sample` for these inputs.

**Review by matthewtsmith — CHANGES_REQUESTED** [review]

File: convert_pkg/package.yaml (hunk 2 above)
Comment (alexisandreason) [reply convert_pkg/package.yaml]: updated

File: convert_pkg/package.yaml (hunk 1 above)
Comment (alexisandreason) [reply convert_pkg/package.yaml]: updated

**Review by matthewtsmith — APPROVED** [review]

**Pull request merged**